tests/test_arguments.py

```diff
@@ -251,13 +245,37 @@ def cmd(arg):
     assert "Missing argument 'ARG'." in result.output
 
 
```

Comment:
Add comment for what this is testing so you can just look at the comment without having to read whole test.

Reply:
Done in https://github.com/pallets/click/pull/3030/commits/ae20270b8764e67fccdf6716e2b852febab9d05c